users/src/test/java/org/spine3/users/UserAccountsShould.java

```diff
@@ -32,6 +32,9 @@ public void return_absent_for_google_uid_if_no_linked_account_added() {
         assertFalse(UserAccounts.getGoogleUid(UserAccount.getDefaultInstance()).isPresent());
     }
 
+    @SuppressWarnings("ConstantConditions")  /* As the {@code DUNBARS_NUMBER} value may change
+                                                a bit, it's still required to stay in certain
+                                                bounds */
```

Comment:
May it? Dunbar's number __is__ declared as a number _n_ which lies between 100 and 250. Though I don't think this particular `Java` constant is going to change with time unless some extraordinary situations which will still allow us to modify the test.
Anyway, the suppression is caused not by the fact that the Dunbar's number is unstable, but with the fact that we check a value of a constant. I think we need no explanation at all with this suppression since it's obvious.
But it's still worth mentioning why do we use exactly the `[100; 250]` bounds.